config/cache.php

```diff
@@ -74,6 +74,6 @@
     |
     */
 
-    'prefix' => 'laravel',
```

Comment:
I think I'd rather keep this one as-is; if I remember correctly we cache quite a bit. No need to bust the cache for this.